blue/readme.md

```diff
@@ -6,11 +6,11 @@
 ## *Modeling to Learn* Links
 
 1. [mtl.how/live](https://www.mtl.how/live) - _Modeling to Learn_ Live Sessions - Adobe Connect Room
-2. [mtl.how/data_test](https://www.mtl.how/data_test) - Team Data User Interface - **Internal for VHA Providers Only
-3. [mtl.how/sim_test](https://www.mtl.how/sim_test) - Simulation User Interface for Teams in _MTL Live_
+2. [mtl.how/data](https://www.mtl.how/data) - Team Data User Interface - **Internal for VHA Providers Only
+3. [mtl.how/sim](https://www.mtl.how/sim) - Simulation User Interface for Teams in _MTL Live_
```

Comment:
Links should remain mtl.how/data_test and mtl.how/sim_test since TEST branch is used for MTL Learners in Test instances. 

@jamesmrollins @mnallajerla We will need an action or edit/scale FNR to edit these links into the Production link when we merge to Master branch.

Reply:
FYI @mnallajerla 

@lijenn - Yes, I see your point.  This is why having learners in Test is problematic.  I found all instances of test links and changed them in this branch. So I recommend after the merge goes through to PROD this time, we double back and change the Test branch back to having links with _test, right?  

When we scale up to an action handling this, then I can automate it to change when we merge with Master.

Does that make sense?

Reply:
@jamesmrollins Makes sense to me!